.github/workflows/install.yaml

```diff
@@ -0,0 +1,71 @@
+name: install
+on:
+  push:
+    branches: '**'
+  pull_request:
+    branches:
+      - 'master'
+      - 'release-[0-9]+.[0-9]+'
+
+jobs:
+  unixish:
+    name: ${{ matrix.os }} ${{ matrix.runner }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - runner: ubuntu-20.04
+            os: linux
+          - runner: macos-10.15
+            os: osx
+    runs-on: ${{ matrix.runner }}
+    if: github.event.pull_request.draft == false
+    steps:
+      - uses: actions/checkout@v2
+      
+      - name: Install dependencies for Linux
+        if: matrix.os == 'linux'
+        run: |
+          sudo apt-get install -y gettext libtool libtool-bin autoconf automake cmake g++ pkg-config unzip build-essential
+          sudo bash ./utils/bin/install-latest-neovim
```

Comment:
We should not specify bash, if bash is required to run the script, then we should update the shebang

Reply:
Yeah, I think we should update it since we have `bash` in the readme for installation as well.

Reply:
#803 has addressed this already

Reply:
Then we only need to do `sudo ./utils/bin/install-latest-neovim`, right ?

Reply:
While reviewing #803 I noticed a typo in the shebang of the installer, which is why you need to specify bash here.